_posts/2024-03-20-infinispan-15-tracing.adoc

```diff
@@ -0,0 +1,135 @@
+---
+layout: blog
+title: Infinispan 15 Tracing Improvement
+permalink: /blog/:year/:month/:day/infinispan-15-tracing
+date: '2024-03-20T9:00:00.000-00:00'
+author: fax4ever
+tags: [ "opentelemetry", "tracing" ]
+---
+
+= Infinispan 15 tracing news
+
+In the previous version of Infinispan https://infinispan.org/blog/2022/07/18/infinispan-14-opentelemetry-tracing[we introduced the OpenTelemetry Tracing capability].
+With the new version we evolved the integration in order to:
+
+* Allow to configure it both globally and on-cache basis using the standard Infinispan configuration.
+* Trace more event types: cluster, cross-site, cache-store and security.
+* Allow to select and configure, also at runtime, the span types that we want to produce and send to the collector.
+
+All the source code we're presenting on this article is taken from https://github.com/fax4ever/infinispan-demo/tree/main/infinispan-15-tracing[this demo project].
+
+== Configure tracing
+
+=== Global configuration
+
+To enable tracing globally you need as usual to define the `collector-endpoint`, but instead of using a system property
```

Comment:
To enable tracing for globally, you define the `collector-endpoint` as usual. Instead of relying on a system property, you can now do this directly in the Infinispan Server configuration: